library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java

```diff
@@ -271,6 +281,10 @@ public void transformPage(View page, float position) {
                     -1
             );
 
+            titleChanger.setOrientation(
+                    a.getInteger(R.styleable.MaterialCalendarView_mcv_titleAnimationOrientation,
+                            HORIZONTAL));
```

Comment:
I don't know if default should be `HORIZONTAL`; people who update the library will see this effect and not be sure why it's different than before